flask_security/core.py

```diff
@@ -342,7 +361,16 @@ def _get_state(app, datastore, anonymous_user=None, **kwargs):
         _unauthorized_callback=None
     ))
 
-    for key, value in _default_forms.items():
+    ident_attrs = app.config.get(
```

Comment:
Please don't shorten the variable names - I would prefer `identity_attributes` for readability purposes.